mtcnn_pytorch/src/align_trans.py

```diff
@@ -246,10 +246,10 @@ def warp_and_crop_face(src_img,
     """
 
     if reference_pts is None:
-        if crop_size[0] == 96 and crop_size[1] == 112:
+        if crop_size is not None and crop_size[0] == 96 and crop_size[1] == 112:
```

Comment:
When will we meet the situation that crop_size is not None?

Reply:
You mean when will it be None?